test/db/formats/elf/loongarch

```diff
@@ -0,0 +1,107 @@
+NAME=ELF: loongarch.elf
+FILE=bins/elf/analysis/loongarch64-curl
+ARGS=-A
+CMDS=?v entry0
+EXPECT=<<EOF
+0xa128
+EOF
+RUN
+
+NAME=ELF: loongarch.elf
+FILE=bins/elf/analysis/loongarch64-curl
```

Comment:
please make a PR to the testbins repository with those binaies so the testsuite can run

Reply:
ok,I submitted the test file before I submitted this patch
https://github.com/radareorg/radare2/pull/19515